src/fusion_power_plant.cc

```diff
@@ -57,19 +61,43 @@ void FusionPowerPlant::EnterNotify() {
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 void FusionPowerPlant::Tick() {
   //pseudocode implementation of Tick():
-  
-  if (CheckOperatingConditions()) {
+
+
+  //State boundary
+  //Changes from "Newly Built" to "Shut Down"
+  //Conditions: initial_startup_condition_met == True
+  //Requirements:
+  //tritium_storage.quantity() >= startup_inventory_requirement (state variable)
+  //Fuel is of correct composition
+  if (!initial_startup_condition_met) { 
+    CheckInitialStartupCondition()
```

Comment:
This test should just be part of `CheckOperatingConditions()`.  Not sure why we need to keep track of the `initial_startup_condition_met`  if we can always query it with `CheckInitialStartupCondtion()`?

Reply:
See my earlier comment, but just to summarize: we need to meet a specific "startup condition" to start operating the first time, but then shouldn't need to meet that condition ever again, but it's important always that we've met that condition that one time.

Reply:
What happens if we become completely depleted of tritium, do we enter this state again? Is that possible?

Reply:
The reason for the different startup condition is to initially saturate all the parts on the plant with tritium (so far as I understand it, at least), so if the plant runs fully out of tritium during operation we shouldn't need to re-enter this state since presumably the plant is already fully, or close to fully, saturated. I can imagine an edge case where a plant would go offline for like 30 years and then maybe need to re-saturate since a significant amount of the saturated tritium has decayed, but a. that feels like a whole different can of worms from the "normal" envisioned use case of this, and b. I feel like, similar to mothballing a fission plant for 30 years, you wouldn't just be trying to flip the switch on it coming back up again and sticking fresh fuel into the facility. That'd be a whole different refurbishment operation which I think is maybe outside the scope of what we're trying to model. 

Thus, we shouldn't ever need to re-enter this state again, and therefore once the facility moves out of this state, there is no way to move back into it.

Reply:
So can't we use the sequestered tritium inventory as a flag for this?

Reply:
we can. I will do that.

Reply:
Were you still planning to make this change, @dean-krueger ?